core/src/main/scala/org/apache/spark/rdd/PairRDDFunctions.scala

```diff
@@ -1014,7 +1015,7 @@ class PairRDDFunctions[K, V](self: RDD[(K, V)])
    * Output the RDD to any Hadoop-supported file system, using a Hadoop `OutputFormat` class
    * supporting the key and value types K and V in this RDD.
    *
-   * Note that, we should make sure our tasks are idempotent when speculation is enabled, i.e. do
+   * @note We should make sure our tasks are idempotent when speculation is enabled, i.e. do
```

Comment:
Ah, I just checked that It seems both

```
We should make sure our tasks are idempotent when speculation is enabled, i.e. do
not use output committer that writes data directly.
```
and
``` 
There is an example in https://issues.apache.org/jira/browse/SPARK-10063 to show the bad
result of using direct output committer with speculation enabled.
```

are relevant (it seems both are related with a direct output committer). 

Also, the original documentation also concatenates both sentences.

![2016-11-18 1 05 24](https://cloud.githubusercontent.com/assets/6477701/20418388/605d112e-ad90-11e6-901c-403acdd26ee6.png)
![2016-11-18 1 05 55](https://cloud.githubusercontent.com/assets/6477701/20418387/605cc19c-ad90-11e6-82e0-afd576766010.png)

So. let me leave it as is if it looks okay.